rust/extractor/src/translate/generated.rs

```diff
@@ -0,0 +1,2185 @@
+use super::base::{TextValue, Translator};
```

Comment:
I think it's nice when generated code starts with a comment like `generated code, do not edit`.

Reply:
Yes, good idea; I also added some `linguist-generated` entries to `.gitattributes`